### PR TITLE
Consolidate Craco Config & CSS file styling mechanisms into LESS files

### PR DESCRIPTION
**Pull request by PaulMEdwards**
There are currently multiple methods of defining styles used in the application.

This PR lays the groundwork to consolidate all of the various styles into Ant Design's LESS mechanism and does so for the Craco Config & CSS files. A subsequent PR will address the style attributes within individual JS files.

 [Consolidate ALL styling mechanisms into LESS files](https://trello.com/c/quoDQaOZ/46-consolidate-all-styling-mechanisms-into-less-files)

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[Consolidate all Styling mechanisms into Ant Design's LESS](https://trello.com/c/quoDQaOZ/46-consolidate-all-styling-mechanisms-into-ant-designs-less)

File: package.json [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
-    "@craco/craco": "^6.1.1",
+    "@craco/craco": "^6.1.2",
     "@stripe/react-stripe-js": "^1.4.0",
     "@stripe/stripe-js": "^1.13.2",
     "antd": "^4.15.0",
     "axios": "^0.21.1",
-    "craco-less": "^1.17.1",
+    "craco-antd": "^1.19.0",
     "date-fns": "^2.21.1",
+    "dayjs": "^1.10.4",
```
[the remark this answers is not in the document]
Comment (PaulMEdwards) [reply package.json:24]: I added it to remove a warning thrown due to it being an unsatisfied dependency of the Craco Ant Design module.

File: antd.customize.less [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import './src/style.less';
```
[the remark this answers is not in the document]
Comment (PaulMEdwards) [reply antd.customize.less:1]: Will do.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (PaulMEdwards) [reply package.json:24]: I switched from `craco-less` to `craco-antd` which also includes `craco-less` as a dependency.
All other modules added were to quelch unsatisfied dependency warnings thrown after doing that.

File: craco.config.js [hunk 3]
```diff
@@ -15,24 +15,11 @@ module.exports = {
...
-              '@body-background': '#fffff6', //cream
-              '@component-background': '#fffff6', //cream
-              '@font-family': 'Nunito, sans-serif',
-              '@icon-color': '#549bea', //blue
-              '@font-size-base': '18px',
-              '@background-color-base': '#fffff6', //cream
-              '@btn-primary-bg': '#549bea', //blue
-              '@btn-height-base': '50px',
-              '@label-required-color': '#549bea', //blue
-            },
```
Comment (PaulMEdwards) [inline craco.config.js:35]: All these styles moved to `index.less`.

File: src/components/Calendar/Calendar.js [hunk 4]
```diff
@@ -1,15 +1,14 @@
-import {React, useState} from 'react'
+import { React, useState } from 'react'
 import { Calendar, dateFnsLocalizer } from 'react-big-calendar'
-import 'react-big-calendar/lib/css/react-big-calendar.css'
-import './calendar.css'
-import { events ,resourceMap } from './data'
-import {customWeekViewEvent, customResourceViewEvent} from './CustomEvent'
+import '../../less/calendar.less'
```
Comment (PaulMEdwards) [inline src/components/Calendar/Calendar.js:3]: Changing the style import from 2 CSS files to 1 LESS file is the only actual change in this file; everything else is just automagic whitespace changes by the linter.

File: craco.config.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 const WebpackBar = require("webpackbar");
-const CracoLessPlugin = require("craco-less");
+const CracoAntDesignPlugin = require("craco-antd");
```
Comment (PaulMEdwards) [inline craco.config.js:2]: Changed from CracoLessPlugin to CracoAntDesignPlugin which also includes CracoLessPlugin.
Will need to run `yarn install`.

File: src/components/Calendar/assign-computers/computers-list.js [hunk 6]
```diff
@@ -1,46 +1,46 @@
-import React,{useEffect, useState} from 'react'
-import "./computers-list-style.css" 
-import {students} from '../data'
-import {mentors} from '../data'
- 
-const ComputersList = (props)=>{
+import React, { useEffect, useState } from 'react'
+import '../../../less/computers-list-style.less'
```
Comment (PaulMEdwards) [inline src/components/Calendar/assign-computers/computers-list.js:2]: Changing the style import from the CSS file to LESS file is the only actual change in this file; everything else is just automagic whitespace changes by the linter.

File: src/components/search-bar/StudentInfo.js [hunk 7]
```diff
@@ -1,62 +1,61 @@
 import { Button } from 'antd'
-import React from "react";
-import Modal from "./Modal.css"
-
+import React from "react"
+import '../../less/Modal.less'
```
Comment (PaulMEdwards) [inline src/components/search-bar/StudentInfo.js:3]: Changing the style import from the CSS file to LESS file is the only actual change in this file; everything else is just automagic whitespace changes by the linter.

File: src/containers/App.js [hunk 8]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import storeRedux from '../redux/store';
-import '../style.less';
```
Comment (PaulMEdwards) [inline src/containers/App.js:5]: This didn't seem to make any difference, so I removed it.

**Comment by PaulMEdwards** [comment]
[Consolidate ALL styling mechanisms into LESS files](https://trello.com/c/quoDQaOZ/46-consolidate-all-styling-mechanisms-into-less-files)

File: package.json [hunk 9]
```diff
@@ -38,9 +39,11 @@
   },
   "devDependencies": {
```
Comment (Brodt7258) [inline package.json:40]: let's get `eslint-plugin-react` in here.

**Review by Brodt7258 — APPROVED** [review]
great. let's do it.

**Review by MichaelS42 — APPROVED** [review]

**Pull request merged**